R/api.R

```diff
@@ -59,6 +59,31 @@
 #' key_get("R-keyring-test-service", "donaldduck")
 #' if (has_keyring_support()) key_list(service = "R-keyring-test-service")
 #' key_delete("R-keyring-test-service", "donaldduck")
+#'
+#' ## This is interactive using backend_file
+#' library(keyring)
+#' 
+#' ## Set variables to be used in keyring
+#' kr_name <- "my_keyring"
+#' kr_service <- "my_database"
+#' kr_username <- "my_username"
+#' 
+#' ## Create a keyring and add an entry using the variables above
+#' kb <- keyring::backend_file$new()
+#' ## Prompt for the keyring password, used to unlock keyring
+#' kb$keyring_create(kr_name)
+#' ## Prompt for the secret/password to be stored in the keyring
+#' kb$set(kr_service, username=kr_username, keyring=kr_name)
+#' # Encrypt the keyring
```

Comment:
`lock()` does not encrypt the keyring, I believe.

Reply:
thanks, resolved